docs/activities/test-resources/index.md

```diff
@@ -0,0 +1,34 @@
+# Resources for Testing
+
+## Test Environments
+There are a number of Test Instances available and the following table compares their features, the most accurate representation of the Production System are CIN4 and the Model office.
+![A table of test environments and their features](./CMS-environments.xlsx)
+
+## How to get access to a test environment
+
+The process for getting access to them is the same.
+
+Navigate to the service request on CPS ServiceNow link below:
+https://cpsprod1.service-now.com/sp?id=sc_cat_item&sys_id=3371d4e01b074110abed0e1ad34bcb9b
```

Comment:
This might be better as:

```
Go to the [CPS ServiceNow](https://cpsprod1.service-now.com/sp?id=sc_cat_item&sys_id=3371d4e01b074110abed0e1ad34bcb9b) request page.
```

Reply:
Yep, much cleaner, and I've made a small tweak as that link takes you to a Test Account request form, so that is the name I used